tests/conftest.py

```diff
@@ -71,16 +72,18 @@ def app(request):
         SQLALCHEMY_DATABASE_URI=os.environ.get(
             'SQLALCHEMY_DATABASE_URI', 'sqlite:///test.db'
         ),
+        RECORDS_REST_ENDPOINTS=config.RECORDS_REST_ENDPOINTS,
         # No permission checking
         RECORDS_REST_DEFAULT_CREATE_PERMISSION_FACTORY=None,
         RECORDS_REST_DEFAULT_READ_PERMISSION_FACTORY=None,
         RECORDS_REST_DEFAULT_UPDATE_PERMISSION_FACTORY=None,
         RECORDS_REST_DEFAULT_DELETE_PERMISSION_FACTORY=None,
         RECORDS_REST_DEFAULT_SEARCH_INDEX=es_index,
-        SEARCH_INDEX_DEFAULT=es_index,
-        SEARCH_AUTOINDEX=[],
         SEARCH_QUERY_ENHANCERS=[filter_record_access_query_enhancer],
+        SEARCH_AUTOINDEX=[],
```

Comment:
`SEARCH_AUTOINDEX` can be removed too. It is not used by invenio-search anymore. See https://github.com/inveniosoftware/invenio-search/commit/cc05a96062370565085bb1504e0dc4698da6e8c7#diff-f6c2f593b9cb0ffb17852882a9a099b8L185